rebar.config

```diff
@@ -90,3 +92,5 @@
 %% Disable warning_as_errors for redbug to avoid deprecation warnings.
 {overrides, [ {del, redbug, [{erl_opts, [warnings_as_errors]}]}
             ]}.
+
+{dist_node, [ {sname, 'erlang_ls'} ]}.
```

Comment:
Most rebar3 commands (e.g. `ct` will use this to start the node in distributed mode).

Reply:
Do we need this though?

Reply:
I don't recall why I had to add it. It was related to the Mnesia setup. Will check...